src_python/mem.py

```diff
@@ -0,0 +1,36 @@
+def mem(num_of_nodes,num_of_edges,num_of_features,feature_data_type,edge_data_type):
+    
+    edge_mem = 2 * num_of_edges * (edge_data_type/8)
+    node_mem = num_of_nodes * num_of_features * (feature_data_type/8)
+
+    graph_size = (edge_mem + node_mem) / (1024*1024*1024)
```

Comment:
Lets move 1024 to meaningful constant

Reply:
What I do here is convert bytes to giga bytes

Reply:
Ok, then lets keep this as it is.